src/components/drawer-content/DrawerContent.tsx

```diff
@@ -0,0 +1,179 @@
+            {props.previousSemesters.map((previousSemester, index) => {
+                return (
+                    <>
+                        <ListItem
+                            button
+                            selected={selectedIndex === index}
+                            onClick={event =>
+                                handleNestedListItemClick(event, index)
+                            }
+                        >
+                            <ListItemText
+                                primary={previousSemester.name}
+                                style={textStyles}
+                            />
+                            {open ? <ExpandLess /> : <ExpandMore />}
+                        </ListItem>
+                        <Divider />
+                        <Collapse in={open} timeout="auto" unmountOnExit>
+                            <List component="div" disablePadding>
+                                {previousSemester.courses.map(
+                                    (course, index) => {
+                                        return (
+                                            <>
+                                                <ListItem
+                                                    button
+                                                    className={classes.nested}
+                                                >
+                                                    <ListItemText
+                                                        style={textStyles}
+                                                        primary={course}
+                                                    />
+                                                </ListItem>
+                                                <Divider />{' '}
+                                            </>
+                                        )
+                                    },
+                                )}
+                            </List>
+                        </Collapse>
+                    </>
+                )
+            })}
+        </List>
+    )
+}
+
+type PreviousSemesters = { name: string; courses: string[] }[]
+
+export const DrawerContent: React.FC = () => {
+    const currentSemester = 'Fall 2019'
+    const courses = user1.courses.map(x => x.name)
+    const previousSemesters: PreviousSemesters = [
+        {
+            name: 'Fall 2018',
+            courses: [
+                'CSC 106',
+                'CSC 110',
+                'HINF 130',
+                'PSYC 100A',
+                'MATH 109',
+            ],
+        },
+        {
+            name: 'Spring 2018',
+            courses: ['CSC 115', 'PSYC 100B', 'MATH 100', 'MATH 122'],
+        },
+    ]
+
+    const gpa = '3.2'
+
+    //  Things to change:
+    //      On any nested loop click, all nested lists open and close -> should only open and close specific nested list
+
+    return (
+        <div id="drawer_content" style={drawerContent}>
```

Comment:
Once you change the way the styling is written above, this is the change you need down here
```suggestion
        <div id="drawer_content" className={classes.drawerContent}>
```